pkg/streamformatter/streamformatter_test.go

```diff
@@ -106,7 +106,7 @@ func TestAuxFormatterEmit(t *testing.T) {
 	sampleAux := &struct {
 		Data string
 	}{"Additional data"}
-	err := aux.Emit(sampleAux)
+	err := aux.Emit("", sampleAux)
```

Comment:
This is a bit awkward (ID being optional)

Reply:
ID was added, most of the time ID is not set (for regular stream output). The whole streamformatter thing is awkward.